stac-example/src/main/scala/geotrellis/stac/raster/StacAsset.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.stac.raster
+
+import geotrellis.stac._
+import com.azavea.stac4s.{StacItem, StacItemAsset}
+import geotrellis.proj4.CRS
+import geotrellis.raster.{CellSize, Dimensions, GridExtent, RasterExtent}
+import geotrellis.stac.extensions.proj.ProjTransform
+import geotrellis.vector.{Extent, Geometry}
+
+case class StacAsset(itemAsset: StacItemAsset, item: StacItem) {
```

Comment:
Can you use `ItemAssetExtension[Proj]` for this instead? That will keep this asset and existing item assets in sync. https://github.com/azavea/stac4s/blob/master/modules/core/shared/src/main/scala/com/azavea/stac4s/extensions/ItemAssetExtension.scala

Reply:
@jisantuc what do you mean by that? this one is a case class to describe the _current_ asset and its item metadata. Mb the name is bad?

Reply:
Ah I see I thought it was hanging out to anchor the proj fields, nevermind, this makes sense